### PR TITLE
Remove unused code

### PR DESCRIPTION
**Pull request by wallner**
This pull-request removes some unused code.

### CODE REVIEW

File: src/main/java/org/osiam/resources/controller/UserController.java [hunk 1]
```diff
@@ -154,7 +154,7 @@ private User setLocationUriAndCreateUserForOutput(HttpServletRequest request, Ht
     /*
      * Checks whether the given user was deactivated and performs necessary subsequent actions.
      */
-    private void checkAndHandleDeactivation(String id, User updateUser, HttpServletRequest request) {
+    private void checkAndHandleDeactivation(String id, User updateUser) {
```
Comment (wallner) [inline src/main/java/org/osiam/resources/controller/UserController.java]: Codacy Issue found: [Avoid unused private methods such as 'checkAndHandleDeactivation(String,User)'.](https://www.codacy.com/app/OSIAM/osiam/file/1895565549/issues/source?bid=3080997&fileBranchId=3093853#l157)

**Comment by wallner** [comment]
Please review #120 first. I will rebase this one after it's merged.

**Comment by dacrome** [comment]
The unit tests are failing.

**Comment by wallner** [comment]
Indeed they were. I mostly used this pr as a test when working on #120 but I now fixed the tests and this pr is good to go.

**Comment by tkrille** [comment]
Thank you!

**Pull request merged**